pkgs/by-name/as/ascii-draw/package.nix

```diff
@@ -0,0 +1,67 @@
+{ lib
+, python3Packages
+, fetchFromGitHub
+, meson
+, ninja
+, pkg-config
+, gobject-introspection
+, wrapGAppsHook4
+, desktop-file-utils
+, libadwaita
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "ascii-draw";
+  version = "0.3.0";
+  pyproject = false;
+
+  src = fetchFromGitHub {
+    owner = "Nokse22";
+    repo = "ascii-draw";
+    rev = "v${version}";
+    hash = "sha256-vI+j8OuQ3b6La0+7wWeoUtBal24dazlN/T0Bng5TgMo=";
+  };
+
+  # Silently fail autosaving to flatpak directory
+  # Autosaving won't work, but program runs normally otherwise
+  # https://github.com/Nokse22/ascii-draw/issues/31
+  postPatch = ''
+    substituteInPlace src/window.py \
+      --replace-fail "/var/data" ""
```

Comment:
Looks like this turned `/var/data/palettes` into `/palettes`?

Reply:
I'd admit I'm dumb

```
=> ls /
bin  boot  dev  etc  home  lib  lib64  nix  palettes  proc  root  run  srv  sys  tmp  usr  var
```